projects.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<link rel="stylesheet" href="styles/main.css">
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+  </head>
+  <header>
+    <h1>AC GILLETTE</h1>
+    <nav>
+      <a href="index.html">Home</a>
+      <a href="about.html">About</a>
+      <a href="projects.html">Projects</a>
+      <a href="writing.html">Writing</a>
+    </nav>
+  </header>
+  <body>
+    <section class="project" id="one">
+      <h2>PROJECT 1</h2>
+    </section>
+    <section class="project" id="two">
+      <h2>PROJECT 2</h2>
+    </section>
+    <section class="project" id="three">
+      <h2>PROJECT 3</h2>
```

Comment:
The photos were sizes the same but displayed in an interesting manner. I'd love to see links to your other projects. Clearly, have a great creative ascetic.